ganga/GangaCore/GPIDev/Lib/Registry/JobRegistry.py

```diff
@@ -137,7 +137,7 @@ def _killJob(this_job):
                             haveKilled = True
                     if not haveKilled:
                         logger.warning("Job status re-updated after potential force-close")
-                        v.updateStatus()
```

Comment:
This was for updating master jobs which are in the wrong state with subjobs in a monitorable state. so I suppose that the updateMasterJobStatus should just be as compatible but I don't know what the difference between these 2 methods now is